src/g_all_guis.c

```diff
@@ -183,6 +183,31 @@ t_symbol *iemgui_raute2dollar(t_symbol *s)
     return(gensym(buf));
 }
 
+t_symbol *iemgui_put_in_braces(t_symbol *s)
+{
+    const char *s1;
+    char buf[MAXPDSTRING+1], *s2;
+    int i = 0;
+    if (strlen(s->s_name) >= MAXPDSTRING)
+        return (s);
+    for (s1 = s->s_name, s2 = buf; ; s1++, s2++, i++)
+    {
+        if (i == 0)
+        {
+            *s2 = '{';
+            s2++;
+        }           
```

Comment:
please remove the trailing whitespace.

(side note, git has a template pre-commit hook that takes care of this; you probably should enable it in your local clone of the repository...)

Reply:
ok. thanks for the tip